python/graphstorm/config/argument.py

```diff
@@ -748,19 +752,72 @@ def eval_fanout(self):
             # By default use -1 as full neighbor
             return [-1] * self.num_layers
 
+    @property
+    def textual_data_path(self):
+        """ distillation textual data path
+        """
+        if hasattr(self, "_textual_data_path"):
+            return self._textual_data_path
+        return None
+
+    @property
+    def lm_name(self):
+        """ HuggingFace Transformer-based model name for distillation
+        """
+        if hasattr(self, "_lm_name"):
+            return self._lm_name
+        return None
+
+    @property
+    def pre_trained_name(self):
+        """ Name of pre-trained HuggingFace model weights for distillation
+        """
+        if hasattr(self, "_pre_trained_name"):
+            return self._pre_trained_name
+        return None
+
+    @property
+    def max_global_step(self):
+        """ Maximum global size for distillation.
+        """
+        # only needed by distillation
+        if "distill" not in self.configuration["gsf"]:
+            return None
+        if hasattr(self, "_max_global_step"):
+            return self._max_global_step
+        else:
+            # default max global steps
+            return 10000
+
+    @property
+    def max_seq_len(self):
+        """ Maximum sequence length for distillation.
+        """
+        # only needed by distillation
+        if "distill" not in self.configuration["gsf"]:
+            return None
+        if hasattr(self, "_max_seq_len"):
+            return self._max_seq_len
+        else:
+            # default max global steps
```

Comment:
what does this mean? is this global step or sequence length?

Reply:
this is a typo. it's sequence length.